hack/ensure-blockcsi-cryaml.sh

```diff
@@ -21,9 +21,9 @@ set -e
 source hack/common.sh
 
 CSI_CR_PATH="config/manager/${BLOCK_CSI_CR_FILE}"
-CSI_CR_URL="https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/${BLOCK_CSI_RELEASE}/deploy/crds/${BLOCK_CSI_CR_FILE}"
+CSI_CR_URL="https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/${BLOCK_CSI_RELEASE}/config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml"
```

Comment:
Please user variable ${BLOCK_CSI_CR_FILE} instead of file name hardcoded.

Reply:
done